google/cloud/spanner_v1/streamed.py

```diff
@@ -121,7 +119,7 @@ def _consume_next(self):
             if source is not None and source._transaction_id is None:
                 source._transaction_id = metadata.transaction.id
 
-        if response.HasField("stats"):  # last response
+        if type(response).pb(response).HasField("stats"):  # last response
```

Comment:
Does `"stats" in response` fail? `proto.Message` has `__contains__` so I would expect it to work.

https://github.com/googleapis/proto-plus-python/blob/b12971a9265e5f35010677fb0d9e1c0eab43320b/proto/message.py#L472

Reply:
`"stats" in response` works. Updated :+1: